### DISCUSSION TITLE
VSCode - how to config?

### DISCUSSION THREAD

**Discussion by hanusek** [SeaQL/FireDBG.for.Rust#26] [Ideas]
Hello!
I installed it:
```
curl https://raw.githubusercontent.com/SeaQL/FireDBG.for.Rust/main/install.sh -sSf | sh
```
and plugin in VSCode. 
But when I click Run and Debug ("Ctrl+Shift+D")...
[Screencast from 22.12.2023 17:32:07.webm](https://github.com/SeaQL/FireDBG.for.Rust/assets/3663212/93af0002-66c2-42d4-8b5e-07627a5810f9)

**Comment by astrale-sharp** [comment]
Try to also install the (EDIT) CodeLLDB extension, that got me farther!

**Reply by tyt2y3** [reply]
FireDBG is based on CodeLLDB, but each has their own binaries and they don't work together.

**Comment by astrale-sharp** [comment]
The setup is not easy via vscode at the moment :
- I didn't get a message to install prebuilt binaries like the getting started section implied
- I had to install another extension (CodeLLDB) to have the basic functionality working
- tests targets did not appear in the vscode UI and don't seem to be accessible

**Reply by tyt2y3** [reply]
I think the installation might have failed or, did you muted the notifications 
![image](https://github.com/SeaQL/FireDBG.for.Rust/assets/1782664/a5ac5cf7-c204-498f-89d3-71d11bfc282c)
?

What OS and version are you on? If you are on Windows, it only works inside WSL.

**Reply by astrale-sharp** [reply]
Thanks for your reply!
I'm on ubuntu, I must have been confused in the gui and only find the right button after installing CodeLLDB, I uninstalled it and eveything still works indeed :sweat_smile:

**Comment by tyt2y3** [comment]
Replying to OP, you can find the "FIREDBG" panel in the bottom left of the "DEBUG" tab. Yeah, it's probably a bit not-noticable. Second, it'd need to be inside a Rust workspace. So the easiest way to try it out is to clone from the "FireDBG.Rust.Testbench" repository.
